ceph-docker-nightly/build/build

```diff
@@ -1,14 +1,17 @@
 #!/bin/bash
 
 # the following two methods exist in scripts/build_utils.sh
-pkgs=( "pytest" )
+pkgs=( "pytest" "docker-py==1.10.3" )
```

Comment:
The docker API changes very very fast, and I am pinning here because I want to prevent having issues while we try to implement/stabilize